manual/core/integration/README.md

```diff
@@ -315,10 +315,6 @@ are not available on your platform, you can exclude the following dependencies:
   <artifactId>java-driver-core</artifactId>
   <version>${driver.version}</version>
   <exclusions>
-    <exclusion>
-      <groupId>com.github.jnr</groupId>
-      <artifactId>jnr-ffi</artifactId>
-    </exclusion>
```

Comment:
👍 

Nit: the text above mentions "libraries" / "dependencies", it should be singular now.

Reply:
Good catch, I'll fix the references above as well.